### PR TITLE
feat(nextcloud,neon): Add compatibility for server 26 and spreed 16

### PR DESCRIPTION
**Pull request by provokateurin**
Closes https://github.com/nextcloud/neon/issues/571

### CODE REVIEW

**Comment by provokateurin** [comment]
Hm so the CI always kills itself for unknown reasons

**Comment by provokateurin** [comment]
We should try to only build the images that we test against. That will reduce the used images drastically and fix it for now.

**Comment by Leptopoda** [comment]
I don't have any particular opinion but I agree that we should be able to count on semver.
It's also the easiest to fix it rn.

**Comment by provokateurin** [comment]
Rebased onto https://github.com/nextcloud/neon/pull/1216 to check if the failure goes away

File: packages/nextcloud/lib/src/patches/spreed/1-compatibility-17.1.json
Comment (provokateurin) [inline packages/nextcloud/lib/src/patches/spreed/1-compatibility-17.1.json:1]: Uh these changes should be a different patch
